denops/@ddc-sources/skkeleton.ts

```diff
@@ -27,7 +27,7 @@ export class Source extends BaseSource<Params> {
     return inputLength - preEditLength;
   }
 
-  async gather(
+  override async gather(
```

Comment:
For abstract method `override` keyword is not necessary. Please see https://github.com/microsoft/TypeScript/issues/47250 for the reason.

```suggestion
  async gather(
```

Reply:
I have read the issue.
It is not necessary.
But to add override is requested in the issue?

Reply:
Ok, indeed.